src/summary.F90

```diff
@@ -537,46 +528,100 @@ subroutine write_tallies(file_id)
         ! Write name of type
         select case (t%filters(j)%type)
         case(FILTER_UNIVERSE)
-          call write_dataset(filter_group, "type_name", "universe")
+          call write_dataset(filter_group, "type", "universe")
```

Comment:
If you choose to use a dictionary to store filter types, remember to reuse it here.